scipy/sparse/linalg/eigen/lobpcg/lobpcg.py

```diff
@@ -287,7 +287,8 @@ def lobpcg(A, X,
     blockVectorX = X
     blockVectorY = Y
     residualTolerance = tol
-    maxIterations = maxiter
+    if maxiter is None:
+        maxiter = 20
```

Comment:
Just curious - why not `min(n, 20)` now?

Reply:
The solver is not intended to be used for n<20, so `min(n, 20)` is redundant and makes no sense.

Reply:
@rgommers in principle this could break backwards compatibility. But as @lobpcg points out, it doesn't make much sense to use this with n<20 anyway. Is this ok?

Reply:
There is no change of the default, still maxiter=20, so no break of backwards compatibility. It is only moved from the function header, replaced with maxiter=None, but still resulting in the same maxiter=20

Reply:
`min(n, 20)` was only mentioned, incorrectly, in the docstring, now also fixed to 20

Reply:
I see. False alarm.